Lib/venv/__init__.py

```diff
@@ -331,7 +330,7 @@ def setup_python(self, context):
                     if not os.path.islink(path):
                         os.chmod(path, 0o755)
         else:
-            if self.symlinks and not _POSIX_BUILD:
```

Comment:
Not really related to the other change, but seems to work fine here. Or was there some motivation for this?

Reply:
I thought that this change was unnecessary, but I can revert.